fix unit test failure on Windows

```
C:\Users\xiazhang\go\src\github.com\kubernetes-csi\csi-proxy>go test -v -race ./internal/...
?       github.com/kubernetes-csi/csi-proxy/internal/os/disk    [no test files]
=== RUN   TestPathValid
--- PASS: TestPathValid (0.51s)
PASS
ok      github.com/kubernetes-csi/csi-proxy/internal/os/filesystem      1.604s
?       github.com/kubernetes-csi/csi-proxy/internal/os/smb     [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/os/volume  [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/server     [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/server/disk        [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/server/disk/internal       [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/server/disk/internal/v1alpha1      [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/server/disk/internal/v1beta1       [no test files]
# github.com/kubernetes-csi/csi-proxy/internal/server/volume [github.com/kubernetes-csi/csi-proxy/internal/server/volume.test]
internal\server\volume\server_test.go:51:2: not enough arguments to return
        have (number, number, nil)
        want (int64, int64, int64, error)
internal\server\volume\server_test.go:99:29: cannot use volAPI (type *fakeVolumeAPI) as type API in argument to NewServer:
        *fakeVolumeAPI does not implement API (missing GetVolumeDiskNumber method)
=== RUN   TestMkdirWindows
E0907 09:50:11.894433    5124 server.go:153] failed validatePathWindows path: C:\foo\bar is not within context path: C:\var\lib\kubelet\pods
E0907 09:50:11.896433    5124 server.go:153] failed validatePathWindows path: C:\foo\bar is not within context path: C:\var\lib\kubelet\plugins
E0907 09:50:11.897433    5124 server.go:153] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\plugins\csi-plugin\pv1:foo
E0907 09:50:11.897433    5124 server.go:153] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\pods\pv1/foo
E0907 09:50:11.897433    5124 server.go:153] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\plugins\csi-plugin\pv1*foo
E0907 09:50:11.897433    5124 server.go:153] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\pods\pv1?foo
E0907 09:50:11.898434    5124 server.go:153] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\plugins\csi-plugin|pv1\foo
E0907 09:50:11.898434    5124 server.go:153] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\pods\pv1\..\..\..\system32
E0907 09:50:11.898434    5124 server.go:153] failed validatePathWindows invalid character \ at beginning of path: \\csi-plugin\..\..\..\system32
E0907 09:50:11.898434    5124 server.go:153] failed validatePathWindows not an absolute Windows path: pv1\foo
--- PASS: TestMkdirWindows (0.01s)
    server_test.go:135: test case: path outside of pod context with pod context set
    server_test.go:135: test case: path inside pod context with pod context set
    server_test.go:135: test case: path outside of plugin context with plugin context set
    server_test.go:135: test case: path inside plugin context with plugin context set
    server_test.go:135: test case: path with invalid character `:` beyond drive letter prefix
    server_test.go:135: test case: path with invalid character `/`
    server_test.go:135: test case: path with invalid character `*`
    server_test.go:135: test case: path with invalid character `?`
    server_test.go:135: test case: path with invalid character `|`
    server_test.go:135: test case: path with invalid characters `..`
    server_test.go:135: test case: path with invalid prefix `\`
    server_test.go:135: test case: relative path
=== RUN   TestRmdirWindows
E0907 09:50:11.899433    5124 server.go:175] failed validatePathWindows path: C:\foo\bar is not within context path: C:\var\lib\kubelet\pods
E0907 09:50:11.899433    5124 server.go:175] failed validatePathWindows path: C:\foo\bar is not within context path: C:\var\lib\kubelet\plugins
E0907 09:50:11.900432    5124 server.go:175] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\plugins\csi-plugin\pv1:foo
E0907 09:50:11.900432    5124 server.go:175] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\pods\pv1/foo
E0907 09:50:11.900432    5124 server.go:175] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\plugins\csi-plugin\pv1*foo
E0907 09:50:11.900432    5124 server.go:175] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\pods\pv1?foo
E0907 09:50:11.900432    5124 server.go:175] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\plugins\csi-plugin|pv1\foo
E0907 09:50:11.901432    5124 server.go:175] failed validatePathWindows path contains invalid characters: C:\var\lib\kubelet\pods\pv1\..\..\..\system32
E0907 09:50:11.901432    5124 server.go:175] failed validatePathWindows invalid character \ at beginning of path: \\csi-plugin\..\..\..\system32
E0907 09:50:11.901432    5124 server.go:175] failed validatePathWindows not an absolute Windows path: pv1\foo
--- PASS: TestRmdirWindows (0.00s)
    server_test.go:253: test case: path outside of pod context with pod context set
    server_test.go:253: test case: path inside pod context with pod context set
    server_test.go:253: test case: path outside of plugin context with plugin context set
    server_test.go:253: test case: path inside plugin context with plugin context set
    server_test.go:253: test case: path with invalid character `:` beyond drive letter prefix
    server_test.go:253: test case: path with invalid character `/`
    server_test.go:253: test case: path with invalid character `*`
    server_test.go:253: test case: path with invalid character `?`
    server_test.go:253: test case: path with invalid character `|`
    server_test.go:253: test case: path with invalid characters `..`
    server_test.go:253: test case: path with invalid prefix `\`
    server_test.go:253: test case: relative path
PASS
ok      github.com/kubernetes-csi/csi-proxy/internal/server/filesystem  2.545s
?       github.com/kubernetes-csi/csi-proxy/internal/server/filesystem/internal [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/server/filesystem/internal/v1alpha1        [no test files]
=== RUN   TestNewSmbGlobalMapping
E0907 09:50:11.893431   23044 server.go:39] remote path is empty
--- PASS: TestNewSmbGlobalMapping (0.00s)
PASS
ok      github.com/kubernetes-csi/csi-proxy/internal/server/smb 2.436s
?       github.com/kubernetes-csi/csi-proxy/internal/server/smb/internal        [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/server/smb/internal/v1alpha1       [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/server/types       [no test files]
FAIL    github.com/kubernetes-csi/csi-proxy/internal/server/volume [build failed]
?       github.com/kubernetes-csi/csi-proxy/internal/server/volume/internal     [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/server/volume/internal/v1alpha1    [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/server/volume/internal/v1beta1     [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/shared/disk        [no test files]
?       github.com/kubernetes-csi/csi-proxy/internal/utils      [no test files]
FAIL
```

--- a/internal/server/volume/server_test.go
+++ b/internal/server/volume/server_test.go
@@ -47,7 +47,17 @@ func (volumeAPI *fakeVolumeAPI) ResizeVolume(volumeID string, size int64) error 
 	return nil
 }
 
-func (volumeAPI *fakeVolumeAPI) VolumeStats(volumeID string) (int64, int64, int64, error) {
+func (volumeAPI *fakeVolumeAPI) GetVolumeDiskNumber(volumeID string) (int64, error) {
+	return -1, nil
+
+}
+
+func (volumeAPI *fakeVolumeAPI) GetVolumeIDFromMount(mount string) (string, error) {
+	return "id", nil
+
+}
+
+func (volumeAPI *fakeVolumeAPI) VolumeStats(volumeID string) (int64, int64, error) {
 	return -1, -1, nil
 }
 

